src/commands/ext-dev-emulators-exec.ts

```diff
@@ -1,11 +1,12 @@
+import * as clc from "cli-color";
+
 import { checkMinRequiredVersion } from "../checkMinRequiredVersion";
 import { Command } from "../command";
-
+import { FirebaseError } from "../error";
 import * as commandUtils from "../emulator/commandUtils";
-import * as optionsHelper from "../extensions/emulator/optionsHelper";
 
 module.exports = new Command("ext:dev:emulators:exec <script>")
-  .description("emulate an extension, run a test script, then shut down the emulators")
+  .description("deprecated")
```

Comment:
consider wording something like

```suggestion
  .description(`deprecated: please use ${localInstallCommand}`)
```